openpype/hosts/traypublisher/plugins/create/create_csv_ingest.py

```diff
@@ -0,0 +1,682 @@
+                    csv_products = csv_data[context_asset_name]["products"]
+                    if pre_product_name not in csv_products:
+                        csv_products[pre_product_name] = {
+                            "task_name": task_name,
+                            "variant": variant,
+                            "product_type": product_type,
+                            "version": version,
+                            "representations": {
+                                filename: representation_data,
+                            },
+                        }
+                    else:
+                        csv_representations = \
+                            csv_products[pre_product_name]["representations"]
+                        if filename in csv_representations:
+                            raise ValueError(
+                                f"Duplicate filename '{filename}' in csv file."
+                            )
+                        csv_representations[filename] = representation_data
+
+        return csv_data
+
+    def _get_representation_row_data(self, row_data):
+        """Get representation row data"""
+        # Get required columns first
+        file_path = self._get_row_value_with_validation(
+            "File Path", row_data)
+        frame_start = self._get_row_value_with_validation(
+            "Frame Start", row_data)
+        frame_end = self._get_row_value_with_validation(
+            "Frame End", row_data)
+        handle_start = self._get_row_value_with_validation(
+            "Handle Start", row_data)
+        handle_end = self._get_row_value_with_validation(
+            "Handle End", row_data)
+        fps = self._get_row_value_with_validation(
+            "FPS", row_data)
+
+        # Get optional columns
+        thumbnail_path = self._get_row_value_with_validation(
+            "Thumbnail", row_data)
+        colorspace = self._get_row_value_with_validation(
+            "Colorspace", row_data)
+        comment = self._get_row_value_with_validation(
+            "Version Comment", row_data)
+        repre = self._get_row_value_with_validation(
+            "Representation", row_data)
+        slate_exists = self._get_row_value_with_validation(
+            "Slate Exists", row_data)
+        repre_tags = self._get_row_value_with_validation(
+            "Representation Tags", row_data)
+
+        # convert tags value to list
+        tags_list = copy(self.representation_config["default_tags"])
+        if repre_tags:
+            tags_list = []
+            tags_delimiter = self.representation_config["tags_delimiter"]
+            # strip spaces from repre_tags
+            if tags_delimiter in repre_tags:
+                tags = repre_tags.split(tags_delimiter)
+                for _tag in tags:
+                    tags_list.append(("".join(_tag.strip())).lower())
+            else:
+                tags_list.append(("".join(tags.strip())).lower())
```

Comment:
maybe should be just repre_tags ?
`tags_list.append(repre_tags)`

Reply:
fixed in https://github.com/ynput/ayon-core/pull/100